analysis/df_epinow.R

```diff
@@ -0,0 +1,44 @@
+# author: Daniel Câmara
+# date: March 3rd, 2022
+
+# libraries used
+library(tidyverse)
+library(RPostgres)
+library(dotenv)
+
+# source for connection
+source(file = "connection.R")
```

Comment:
There should be a function defined inside `connection.R` that returned the connection object.

Reply:
Done in the new push.